web-terminal/instance/base/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
I have only used the web terminal with the default instance, which I don't believe, you need to create.

I know you have been doing cool stuff with the web terminal operator, but I'm not sure I understand enough of what this instance overlay does above and beyond the default one.  Can you add some documentation about them?